.vscode/launch.json

```diff
@@ -0,0 +1,15 @@
+{
+    // Use IntelliSense to learn about possible attributes.
+    // Hover to view descriptions of existing attributes.
+    // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
+    "version": "0.2.0",
+    "configurations": [
+        {
+            "type": "chrome",
+            "request": "launch",
+            "name": "Launch Chrome against localhost",
+            "url": "http://localhost:3000",
+            "webRoot": "${workspaceFolder}/app"
+        }
+    ]
+}
```

Comment:
Not sure this is intended to have in this PR. I think those kind of files should remain personal no ? Everyone has their favorite browser etc..

Reply:
This is kind of a ghost from the PR #1251 where I'm adding this anyways.  Could remove it i guess, from this PR just to keep it tidy and leave the discussion on whether its important to include it in the other one 🤷‍♂️